testing/Dockerfile.sdk.test

```diff
@@ -1,6 +1,13 @@
-FROM node:18-alpine@sha256:44aaf1ccc80eaed6572a0f2ef7d6b5a2982d54481e4255480041ac92221e2f11
+FROM node:19.6.1-slim@sha256:a1ba21bf0c92931d02a8416f0a54daad66cb36a85d2b73af9d73b044f5f57cfc
```

Comment:
Should we use a node 18 slim image?

Reply:
@chase-45

Reply:
I don't think anything about the CI process is sensitive to 19 vs 18, so either should be acceptable